<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addstadium</class>
 <widget class="QWidget" name="addstadium">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>160</width>
     <height>430</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="stadiumName"/>
    </item>
    <item>
     <widget class="QLineEdit" name="teamName"/>
    </item>
    <item>
     <widget class="QLineEdit" name="majorLeague"/>
    </item>
    <item>
     <widget class="QLineEdit" name="streetAdd"/>
    </item>
    <item>
     <widget class="QLineEdit" name="city"/>
    </item>
    <item>
     <widget class="QLineEdit" name="state"/>
    </item>
    <item>
     <widget class="QLineEdit" name="zip"/>
    </item>
    <item>
     <widget class="QLineEdit" name="boxOffice"/>
    </item>
    <item>
     <widget class="QLineEdit" name="monthOpened"/>
    </item>
    <item>
     <widget class="QLineEdit" name="dayOpened"/>
    </item>
    <item>
     <widget class="QLineEdit" name="yearOpened"/>
    </item>
    <item>
     <widget class="QLineEdit" name="capacityOne"/>
    </item>
    <item>
     <widget class="QLineEdit" name="capacityOneDes"/>
    </item>
    <item>
     <widget class="QLineEdit" name="capacityTwo"/>
    </item>
    <item>
     <widget class="QLineEdit" name="capacityTwoDes"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>160</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Stadium Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Team Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>League</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Street Address</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>City</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>State</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Zip</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Box Office Number</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Month</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Day</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Year</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Capacity One</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Capacity One Description</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Capacity Two</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Capacity Two Description</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>480</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>480</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
